formstack-baton-requests/src/main/scala/com/gu/identity/formstackbatonrequests/aws/S3Client.scala

```diff
@@ -13,9 +14,13 @@ case class S3CompletedPathFound(resultLocations: List[String]) extends S3StatusR
 case class S3FailedPathFound() extends S3StatusResponse
```

Comment:
again this class is called S3Client. I personally think that the methods should not have s3 in them. we could read it as s3Client.S3FailedPathFound or s3Client.FailedPathFound

Reply:
agree - done